### PR TITLE
Sort events according to date by default on dashboard

### LINKED ISSUE #5175: Event Dashboard and Event Admin Dashboard: Sort Events according to Dates by Default
To speed up organizers and admins usage of the platform please change the dashboard sorting:
* By default sort table according to oldest date first compare: https://eventyay.com/events/live?sort_by=starts-at&sort_dir=DSC
* Also sort in the same way on the admin dashboard: https://eventyay.com/admin/events/live?sort_by=starts-at&sort_dir=DSC

Screenshot from 2020-09-29 11-55-17

### PR DESCRIPTION
**Pull request by sahilsaha7773**
Fixes #5175 

#### Short description of what this resolves:

Enchances the dashboard experience by sorting the events according to date by default.

#### Changes proposed in this pull request:

- Added default sorting to the start-date column of events table.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes

### CODE REVIEW

File: app/components/tables/headers/sort.js [hunk 1]
```diff
@@ -15,6 +15,12 @@ export default class Sort extends Component {
         return 'caret down';
       }
     }
+    else {
+      //default sorting for starts-at column
+      if(kebabCase(this.column.valuePath)=='starts-at'){
```
Comment (niranjan94) [inline app/components/tables/headers/sort.js:20]: Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483&bid=20437942)

File: app/components/tables/headers/sort.js (hunk 1 above)
Comment (niranjan94) [inline app/components/tables/headers/sort.js:20]: Codacy found an issue: [Operator '==' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)

Comment (niranjan94) [inline app/components/tables/headers/sort.js:20]: Codacy found an issue: [Missing space before opening brace.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)

Comment (niranjan94) [inline app/components/tables/headers/sort.js:20]: Codacy found an issue: [Expected space(s) after "if".](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)

File: app/components/tables/headers/sort.js [hunk 2]
```diff
@@ -35,6 +41,13 @@ export default class Sort extends Component {
           sortDir : null
         });
       }
+      else {
+        //default sorting for starts-at
```
Comment (niranjan94) [inline app/components/tables/headers/sort.js:45]: Codacy found an issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)

File: app/components/tables/headers/sort.js [hunk 3]
```diff
@@ -15,6 +15,12 @@ export default class Sort extends Component {
         return 'caret down';
       }
     }
+    else {
+      //default sorting for starts-at column
```
Comment (niranjan94) [inline app/components/tables/headers/sort.js:19]: Codacy found an issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Wrong solution

**Pull request closed without merging**